NativeApp/src/Android/AndroidAppBase.cpp

```diff
@@ -113,27 +127,91 @@ int32_t AndroidAppBase::HandleInput(android_app* app, AInputEvent* event)
 //
 // Process the next main command.
 //
-void AndroidAppBase::HandleCmd(struct android_app* app, int32_t cmd)
+void AndroidAppBase::HandleCmd(struct android_app* android_app, int32_t cmd)
 {
-    AndroidAppBase* eng = (AndroidAppBase*)app->userData;
+    AndroidAppBase* eng = (AndroidAppBase*)android_app->userData;
     switch (cmd)
     {
-        case APP_CMD_SAVE_STATE:
+        case APP_CMD_INPUT_CHANGED:
+#if(LOG_NATIVE_APPLICATION)
+            LOG_ERROR_MESSAGE("APP_CMD_INPUT_CHANGED");
+#endif
+            // is this safe? disable for now
+
+            //pthread_mutex_lock(&android_app->mutex);
+            //if (android_app->inputQueue != NULL) {
+            //    AInputQueue_detachLooper(android_app->inputQueue);
+            //}
+            //android_app->inputQueue = android_app->pendingInputQueue;
+            //if (android_app->inputQueue != NULL) {
+            //    LOG_ERROR_MESSAGE("Attaching input queue to looper");
+            //    AInputQueue_attachLooper(android_app->inputQueue,
+            //                             android_app->looper, LOOPER_ID_INPUT, NULL,
+            //                             &android_app->inputPollSource);
+            //}
+            //pthread_cond_broadcast(&android_app->cond);
+            //pthread_mutex_unlock(&android_app->mutex);
             break;
 
         case APP_CMD_INIT_WINDOW:
-            // The window is being shown, get it ready.
-            if (app->window != NULL)
-            {
+            pthread_mutex_lock(&android_app->mutex);
+#if(LOG_NATIVE_APPLICATION)
+            LOG_ERROR_MESSAGE("APP_CMD_INIT_WINDOW");
```

Comment:
Why error?

Reply:
to make it stand out in the log as it is important for seeing what is going on with the state of the native app internally when debugging the state, since errors have red text in logcat